Sources/PaymentsCore/AnalyticsPayload.swift

```diff
@@ -0,0 +1,93 @@
+import UIKit
+
+struct AnalyticsPayload: Encodable {
```

Comment:
Open to better naming ideas for this top-level type.

Reply:
🤔  would `AnalyticsObject` or `AnalyticsEventData` be clearer? I am also not opposed to Payload, it looks like that's what some other SDKs use as well.

Reply:
I dont mind `AnaliticsPayload` either